lua/bufferline/render.lua

```diff
@@ -138,7 +138,7 @@ local function render()
         icon = number_text .. (#number_text > 1 and '' or ' ')
       else
         local iconChar, iconHl = get_icon(buffer_name, vim.fn.getbufvar(buffer_number, '&filetype'))
-        iconPrefix = status ~= 'Current' and hl('BufferInactive') or hl(iconHl or ('Buffer' .. status))
+        iconPrefix = status ~= 'Current' and hl('Buffer'..status) or hl(iconHl or 'BufferCurrent')
```

Comment:
Now it will infer the highlight group from the status when the group is unknown (`~= 'Current'`), rather than when it is known (`== 'Current'`).

Reply:
This seems to make the icon of visible (but not current) tabs uncolored.

Reply:
You're right. The reason for this is because of what happens when using certain colorschemes with the devicons implementation:

![DevIcons](https://user-images.githubusercontent.com/36409591/99316626-acf2c800-2832-11eb-8c75-4f26b066b989.png)

I'm attempting to sidestep that issue altogether with this. Alternatively we can pursue it with the maintainers of the library in order to restore the functionality

Reply:
The latest commit fixes this:

![cap](https://user-images.githubusercontent.com/36409591/99323841-019d3f80-2841-11eb-8cb1-67323af324d4.png)